app/views/directives/edit-environment-from.html

```diff
@@ -149,8 +152,11 @@
       <div ng-if="$ctrl.overlayPaneEntryDetails.data | size" class="table-responsive scroll-shadows-horizontal">
         <table class="table table-bordered table-bordered-columns config-map-table key-value-table">
           <tbody>
-            <tr ng-repeat="(prop, value) in $ctrl.decodedData">
-              <td class="key">{{prop}}</td>
+            <tr ng-repeat="(key, value) in $ctrl.decodedData">
+              <td class="key">
+                <span ng-if="$ctrl.isEnvVarInvalid(key)" class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: middle;" data-toggle="popover" data-trigger="hover" dynamic-content="{{key}} is not a valid environment variable name and will not be added."></span>
```

Comment:
Better to have these styles in a CSS file somewhere than declaring them inline. I would guess we have a tooltip class somewhere. If not, we should add it. (I know we have inline tooltip styles other places, but I'd like to get out of the habit of doing that since it's not a good practice.)

cc @rhamilto @sg00dwin